### DISCUSSION TITLE
Can i use http instead of grpc in java agent ?

### DISCUSSION THREAD

**Discussion by nanxing137** [apache/skywalking#11248] [Q&A] [answered]
I want use http to send data,To maintain consistency with nginx

**Comment by wu-sheng** [comment, marked as answer]
No, you can't use HTTP.

**Reply by nanxing137** [reply]
But i saw this [ https://github.com/apache/skywalking/issues/6027 ]. It's means that  in the past version I can  use http?

**Reply by wu-sheng** [reply]
That was a 5 years ago thing. I don't think that is pointless.

**Reply by nanxing137** [reply]
thx
